profiles/seccomp/seccomp.go

```diff
@@ -6,11 +6,11 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"runtime"
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/pkg/parsers/kernel"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
-	libseccomp "github.com/seccomp/libseccomp-golang"
```

Comment:
Should be removed from vendor?

Reply:
Also Dockerfile should no longer need aptgetting libseccomp

Reply:
Still need libseccomp to build runc at least.

Reply:
I may push some images for runc and containerd since these rarely change and just consume those in our Dockerfile...